highway.tex

```diff
@@ -24,807 +26,1323 @@
+%
+Flexibility then means that despite these differences in configuration, all the nodes run a single version of the protocol and perform the same actions, only the finality decisions they make depend on the chosen parameters.
+%
+A practical consequence is that nodes with lower security thresholds might reach finality much faster than nodes with higher thresholds, but as long as both these nodes' assumptions are satisfied they finalize the same blocks and stay in agreement.
+    
+Technically, Highway can be categorized as a DAG-based protocol~\cite{moser1999byzantine,zamfir2018casper,baird2016hashgraph,GLSS19}, in which nodes jointly maintain a common history of protocol messages, forming a directed acyclic graph representing the causality order.
+%
+In its design, Highway derives from the CBC-Casper protocol~\cite{zamfir2018casper} and significantly improves upon it by the use of a new finality mechanism, message creation schedule and spam prevention mechanism.
+%
+We believe that the conceptual simplicity of DAG-based protocols along with the desirable practical features of the Highway protocol make it a solid choice for a consensus engine in a proof of stake based blockchain. 
 
-Bitcoin \cite{nakamoto2008bitcoin} introduced a \emph{permissionless} Proof-of-Work consensus algorithm, where the set of participating nodes is continually changing, and anyone can join or leave at any time, and Ethereum \cite{wood2014ethereum} extended this concept from a pure cryptocurrency to a general programmable smart contract platform. Both networks have very high value and correspondingly high requirements for security, due to potentially large incentives for attackers.
 
-Proof-of-Work inherently has extremely high energy requirements \cite{de2018bitcoin}. Proof-of-Stake networks (e.g. \cite{kiayias2017ouroboros}) attempt to address this issue by giving the power to make consensus decisions not to the nodes that contribute the most computing resources, but to the ones that deposit the highest stakes as a collateral. They aspire to provide the same or higher levels of security and performance, while requiring less energy by many orders of magnitude.
 
 
-\subsection*{Our Contribution}
+\section{Our Results}
 
-Highway is a concrete consensus algorithm based on CBC Casper \cite{zamfir2018casper} together with a criterion for finality, both proposed by Daniel Kane. It is live in a partially synchronous network, if the participants have clocks that are synchronized up to an unknown but bounded difference.
+\subsection{Model}
 
-At the heart of Casper we made two design choices that keep it simple and intuitive:
+We consider a system with a fixed set $\mathcal{V}$ of $n$ validators, each of them equipped with public key that is known to other nodes.
+This model matches the scenario of 'permissioned blockchain', but the protocol can be applied to semi-permissionless scenario as well by rotating the set of validators.
```

Comment:
We could follow the [Overleaf recommendations for quotations](https://www.overleaf.com/learn/latex/typesetting_quotations).

Reply:
I changed all quotations to the proper format.